test/test_object.py

```diff
@@ -18,6 +18,65 @@ def test_trailing_slash(self):
         identity = slash_identity.strip(posixpath.sep)
         self.assertEqual(identity, c.identity)
 
+    def test_copy_properties(self):
+        doc = sbol3.Document()
+        root = sbol3.Component('root', sbol3.SBO_DNA)
+        root.name = 'foo'
+        root_copy = root.copy()
+        self.assertEqual(root_copy.name, 'foo')
+
+    def test_copy_child_objects(self):
+        doc = sbol3.Document()
+        root = sbol3.Component('root', sbol3.SBO_DNA)
+        sub1 = sbol3.Component('sub1', sbol3.SBO_DNA)
+        sub2 = sbol3.Component('sub2', sbol3.SBO_DNA)
+        sc1 = sbol3.SubComponent(sub1)
+        sc2 = sbol3.SubComponent(sub2)
+        root.features.append(sc1)
+        root.features.append(sc2)
+        doc.add(root)
+        doc.add(sub1)
+        doc.add(sub2)
+
+        doc2 = sbol3.Document()
+        root_copy = root.copy(target_doc=doc2)
+        self.assertEqual([sc.identity for sc in root.features],
+                         [sc.identity for sc in root_copy.features])
+
+    # def test_import_object_into_new_namespace(self):
```

Comment:
Can this commented out test method either be documented or removed?

If it is for a future capability, let's put in some comments explaining that so we know why it's there. If it served as a model for other tests and is not expected to ever be a pySBOL3 test then let's remove it.

Reply:
OK, I'll remove it for now.  That's a test case ported over from pySBOL2 that deals with namespace replacement.